examples/bridge-app/linux/main.cpp

```diff
@@ -895,22 +886,20 @@ int main(int argc, char * argv[])
     ActionLight3.SetReachable(true);
     ActionLight4.SetReachable(true);
 
-    // Define composed device with two switches
-    ComposedDevice ComposedDevice("Composed Switcher", "Bedroom");
-    DeviceSwitch ComposedSwitch1("Composed Switch 1", "Bedroom", EMBER_AF_SWITCH_FEATURE_LATCHING_SWITCH);
-    DeviceSwitch ComposedSwitch2("Composed Switch 2", "Bedroom",
-                                 EMBER_AF_SWITCH_FEATURE_MOMENTARY_SWITCH | EMBER_AF_SWITCH_FEATURE_MOMENTARY_SWITCH_RELEASE |
-                                     EMBER_AF_SWITCH_FEATURE_MOMENTARY_SWITCH_LONG_PRESS |
-                                     EMBER_AF_SWITCH_FEATURE_MOMENTARY_SWITCH_MULTI_PRESS);
+    // Define composed device with two temperature sensors
+    ComposedDevice ComposedDevice("Composed Device", "Bedroom");
     DevicePowerSource ComposedPowerSource("Composed Power Source", "Bedroom", EMBER_AF_POWER_SOURCE_FEATURE_BATTERY);
 
-    ComposedSwitch1.SetChangeCallback(&HandleDeviceSwitchStatusChanged);
-    ComposedSwitch2.SetChangeCallback(&HandleDeviceSwitchStatusChanged);
+    ComposedTempSensor1.SetMeasuredValue(100);
+    ComposedTempSensor2.SetMeasuredValue(100);
```

Comment:
Please put the SetMeasuredValue calls after the SetChangeCallback calls, for this to work on tip.

Reply:
I changed this so the constructor can set the initial value, so that this set was not needed.